SilMock/Google/Service/Directory/Resource/Members.php

```diff
@@ -15,6 +15,19 @@ public function __construct(?string $dbFile = null)
         parent::__construct($dbFile, 'directory', 'members');
     }
 
+    public function get(string $groupKey, string $memberKey): GoogleDirectory_Member
+    {
+        $members = $this->listMembers($groupKey);
+        $memberList = $members->getMembers();
+        foreach ($memberList as $member) {
+            $memberEmailAddress = mb_strtolower($member->getEmail());
+            if ($memberEmailAddress === mb_strtolower($memberKey)) {
+                return $member;
+            }
+        }
+        throw new Exception('Member not found');
```

Comment:
The docs aren't clear about what the actual Google API does in this case. Do you know if it also throws some kind of exception if the member is not found?

Reply:
True, but given your more current "throw exception unless it works" philosophy seems more modern and in line with Google's current thinking.